OutOfSchool/OutOfSchool.BusinessLogic/Services/Database/WorkshopService.cs

```diff
@@ -157,23 +138,17 @@ public async Task<WorkshopResultDto> CreateV2(WorkshopV2Dto dto)
         _ = dto ?? throw new ArgumentNullException(nameof(dto));
         logger.LogInformation("Workshop creating was started.");
 
+        // TODO: after refactoring the DTOs for the Workshop entities, this method needs to be replaced with the correct mapping
+        await SetIdsToDefaultValue(dto); // This method sets the properties with the Id to the default value.
```

Comment:
We get the WorkshopCreateUpdateDto object from Frontend. It may contain not only null value but also invalid values, so we will have to replace them with the default value in the SetIdsToDefaultValue() method.
To avoid this, we can do it using mapping.